app/src/main/java/com/mapbox/maps/testapp/examples/InsetMapActivity.kt

```diff
@@ -44,7 +45,7 @@ class InsetMapActivity : AppCompatActivity(), OnCameraChangeListener {
     mainMapboxMap = mainMapView.getMapboxMap()
     mainMapboxMap.loadStyleUri(
       styleUri = STYLE_URL
-    ) { mainMapboxMap.addOnCameraChangeListener(this@InsetMapActivity) }
+    ) { mainMapView.getCameraAnimationsPlugin().addOnCameraChangeListener(this@InsetMapActivity) }
```

Comment:
can we expose the `addOnCameraChangeListener` on the `mapboxMap`?

Reply:
It will interfere with current deprecated one.

Reply:
Not relevant anymore.